packages/pictogrammers-1.0.0.yaml

```
@@ -0,0 +1,5 @@
+name: pictogrammers_icons
+description: These icons are from the Pictogrammers project which builds on the free icons released by the Google Material Design project.  See https://materialdesignicons.com/ and https://pictogrammers.com/
```

I think this is valid yaml
```suggestion
description: >
  These icons are from the Pictogrammers project which builds
  on the free icons released by the Google Material Design
  project.  See https://materialdesignicons.com/ and
  https://pictogrammers.com/
```